src/libs/ReportActionsUtils.js

```diff
@@ -383,14 +379,24 @@ function shouldReportActionBeVisibleAsLastAction(reportAction) {
  * @return {Object}
  */
 function getLastVisibleAction(reportID, actionsToMerge = {}) {
-    const actions = _.toArray(lodashMerge({}, allReportActions[reportID], actionsToMerge));
-    const visibleActions = _.filter(actions, (action) => shouldReportActionBeVisibleAsLastAction(action));
+    const updatedActionsToMerge = {};
+    if (actionsToMerge && Object.keys(actionsToMerge).length !== 0) {
+        Object.keys(actionsToMerge).forEach(
+            (actionToMergeID) => (updatedActionsToMerge[actionToMergeID] = {...allReportActions[reportID][actionToMergeID], ...actionsToMerge[actionToMergeID]}),
+        );
+    }
+    const actions = Object.values({
+        ...allReportActions[reportID],
+        ...updatedActionsToMerge,
+    });
```

Comment:
Looks like this change introduced this issue: https://github.com/Expensify/App/issues/28941 fixed in this PR: https://github.com/Expensify/App/pull/29939